src/ChangelogGenerator/Command/GenerateChangelogCommand.php

```diff
@@ -351,4 +367,26 @@ private function getIncludeOpen(InputInterface $input) : bool
 
         return false;
     }
+
+    private function getIncludeDate(InputInterface $input) : bool
```

Comment:
Can we make a common private method and use it for both `getIncludeOpen` and `getIncludeDate`? They are identical except for the option name. Thoughts?

Reply:
If you like i can refactor this command or only merge both functions

Reply:
We should probably refactor so we're not duplicating code.